apps/server/src/stores/__tests__/runtimeState.test.ts

```diff
@@ -323,11 +339,17 @@ describe('roll mode', () => {
         order: ['1', '2', '3'],
       });
 
+      // force update
+      vi.useFakeTimers();
+      await initRundown(rundown, {});
+      vi.runAllTimers();
+
       load(rundown.entries[1] as PlayableEvent, rundown, rundown.order);
       start();
+      const currentOffset = getState().runtime.offset;
       let result = roll(rundown, rundown.order, getState().runtime.offset);
       expect(result).toStrictEqual({ eventId: '1', didStart: false });
-      expect(getState().runtime.offset).toBe(1000);
+      expect(getState().runtime.offset).toBe(currentOffset);
```

Comment:
question: can you clarify why we are keeping the value of `currentOffset` and comparing it after the roll call?
Is the objective to make sure it didnt change?
If so: the name of the variable should be something like `offsetBeforeRoll`, we could also do with a small comment explaining that since it is not explicit in code